packages/l/lame/xmake.lua

```diff
@@ -23,6 +24,15 @@ package("lame")
         import("package.tools.autoconf").install(package, configs)
     end)
 
+    on_install("windows|x86", function (package)
```

Comment:
Does it not support x64 ?

Reply:
current release does not support, because nasm assemble code explicitly requires 32bit layout